### ISSUE TITLE
Hakuneko for Apple silicon M1 Series

### ISSUE THREAD

**Issue by crucknova** [manga-download/hakuneko#4727]
After the long transition of the macbook from x86 to arm, need to convert the application or the build to Arm devices so the app can run natively on the macbook with m1 chips without rosetta, which is another battery and resource consuming process!

**Comment by ma-tecik** [comment]
There are ARM versions for Linux, so I don't think it's that hard, but I'm not sure. I wish someone add that.

**Comment by alfonslm** [comment]
Yes, I doubt that even with the auto update that it would switch from amd64 to arm64

**Comment by akhilyeddula** [comment]
With MacOS 26.4 warning users about non-native applications and MacOS 27 being the last version to support Rosetta 2, this should be prioritized sooner rather than later. 

Seeing that Linux has ARM versions means that porting it to MacOS shouldn't be too hard to my knowledge (of course, I may be incorrect). 

I looked at the documentation and it says this:

> Due to missing arm64 builds of older electron releases for Apple M1, it is necessary to install the Intel version, e.g.

However, this could be theoretically solved by updating from Electron v8 to the newest one, because Electron (v20+) has full arm64 support for macOS. This would probably be a massive undertaking by the team though and might not be worth it since there is a new app in the works.

This is the project where they are rewriting the application and it has a native Apple Silicon build:
https://github.com/manga-download/haruneko
